core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/EntityStatisticsCacheMonitor.scala

```diff
@@ -26,10 +25,14 @@ object EntityStatisticsCacheMonitor {
   sealed trait EntityStatisticsCacheMessage
   case object Sweep extends EntityStatisticsCacheMessage
   case object ScheduleDelayedSweep extends EntityStatisticsCacheMessage
+
+  //Note: Ignored workspaces have a cacheLastUpdated timestamp of 1000ms after epoch
+  val MIN_CACHE_TIME = new Timestamp(1000)
```

Comment:
Is this value exclusively used to identify workspaces whose caches didn't refresh due to error and should now be left alone?  If yes, my pretty strong preference would be not to overload the meaning of `entityCacheLastUpdated` in that fashion.  Let's just add an separate column to the `WORKSPACE_ENTITY_CACHE` to denote those workspaces whose caches aren't refreshing. That approach has the benefit of lowering the cognitive burden and more easily allowing for retries.

Reply:
it is, yes - how would you feel about changing that logic as part of AS-931 instead of in this PR?

Reply:
That's fine.